app/frontend/stylesheets/modules/_colors.scss

```diff
@@ -1,10 +1,13 @@
 // https://color-name-generator.com/ 
 
+$white: #FFF;
```

Comment:
white is already set by bootstrap

Reply:
I realized I'm not even using it either!